compatibility_server/views.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from opencensus.stats import aggregation as aggregation_module
```

Comment:
Give this file, I'd say why not put `Stats` here as well and have everyone use the same global. But this is OK too.

Reply:
The goal of this file was to have a place to define new metrics (view objects), so I think `Stats` doesn't necessarily need to be here.

Reply:
The current separation of responsibilities is kind of odd now.

All of the view are defined here but they are registered in `compatibility_checker_server.py`.

If you moved Stats here then you could write:

```python

STATS = stats_module.Stats()
...
DOCKER_ERROR_VIEW = view_module.View(...)
STATS.view_manager.register_view(DOCKER_ERROR_VIEW)
```

Reply:
I think `Stats` needs to stay in `compatibility_checker_server.py` because it's necessary for calling `_enable_exporter()` which needs to stay in that file as well, and calling that function would look super weird if `Stats` were in `views.py`